db/projects/augur/project.yaml

```diff
@@ -0,0 +1,66 @@
+name: Augur
+start_date: '2014-11-17'
+description: 'Augur is a Decentralized Prediction Market Platform'
+web:
+  - 'http://www.augur.net/'
+assets:
+  - id: rep
+    start_date: '2014-11-17'
+    name: Augur
+    type: token
+    token_standard: ERC-20
+    contract_address: '0xe94327d07fc17907b4db788e5adf2ed424addff6'
+    ledger: ethereum
+    symbol: REP
+    max_supply: 11000000
+    ico_properties:
+      start_date: '2015-08-17'
+      end_date: '2017-10-01'
+      total_cap: 8800000
+      raised: 5000000 USD
+    webids:
+      coinmarketcap: augur
+      etherscan/token: REP
+      cryptocompare: rep
+      cryptocompare/id: '17778'
+      coingecko: augur
+webids:
+  bitcointalk: '860629.0'
+  bitcoinwiki: Augur
+  discord: faud6Fx
+  stackexchange: augur
+  facebook: augurproject
+  github: AugurProject
+  linkedin/company: augur-net
+  telegram: AugurProject
+  twitter: AugurProject
+  reddit: augur
+  youtube/channel: UCnQRWIWIT8ExlegLTajjhiQ
+whitepapers:
+  - title: 'Augur Whitepaper'
+    url: https://github.com/AugurProject/whitepaper
```

Comment:
Link to PDF file
url: https://github.com/AugurProject/whitepaper/blob/master/english/whitepaper.pdf